tests/Fixer/Casing/NativeFunctionTypeDeclarationCasingFixerTest.php

```diff
@@ -211,16 +211,21 @@ public function testFix82(string $expected, string $input): void
         $this->doTest($expected, $input);
     }
 
-    public function provideFix82Cases(): \Generator
+    public function provideFix82Cases(): iterable
     {
-        yield 'return type `false`' => [
-            '<?php class T { public function Foo(object $A): false {return false;}}',
-            '<?php class T { public function Foo(object $A): FALSE {return false;}}',
+        yield 'standalone type `true`' => [
+            '<?php class T { public function Foo(true $A): true {return $A;}}',
+            '<?php class T { public function Foo(True $A): TRUE {return $A;}}',
         ];
 
-        yield 'return type `null`' => [
-            '<?php class T { public function Foo(object $A): null {return null;}}',
-            '<?php class T { public function Foo(object $A): NULL {return null;}}',
+        yield 'standalone type `false`' => [
+            '<?php class T { public function Foo(false $A): false {return $A;}}',
+            '<?php class T { public function Foo(False $A): FALSE {return $A;}}',
+        ];
+
+        yield 'standalone type `null`' => [
+            '<?php class T { public function Foo(null $A): null {return $A;}}',
+            '<?php class T { public function Foo(Null $A): NULL {return $A;}}',
```

Comment:
Can you add one more test with `\null`, `\true` and `\false`?

Reply:
Since we have 2 usages of type in each test, we can use non-backslahed and backslashed version, without adding another test. Would it be acceptable?

Reply:
It should be fine, but I see now we have `fixFunctionArgumentTypes` and `fixFunctionReturnType` methods in that fixer, so let's keep param `$A` and return type as it is and add param `$B` with backslashes type, WDYT?